CHANGES/8968.bugfix

```diff
@@ -0,0 +1 @@
+Added a guard to avoid a rare ``'NoneType' object has no attribute 'delete'`` error.
```

Comment:
Could it be rephrased, to avoid the word guard? E.g. 
```
Fixed ``'NoneType' object has no attribute 'delete'`` error during migration re-runs.
```